charts/sn-platform/templates/tls/tls-certs-internal.yaml

```diff
@@ -75,7 +75,65 @@ spec:
 ---
 {{- end }}
 {{- end }}
-
+{{- if and .Values.tls.broker.enabled (and .Values.istio.enabled (eq .Values.istio.gateway.tls.mode "PASSTHROUGH")) }}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.tls.broker.cert_name }}-gateway-passthrough"
```

Comment:
Rather than creating a new cert, how about just change the dnsNames of the cert `{{ template "pulsar.fullname" . }}-{{ .Values.tls.broker.cert_name }}`?